org.civicrm.reqangular/src/common/angular-date/src/filters/CustomDateFilter.js

```diff
@@ -18,8 +20,18 @@ module.exports = function ($filter, DateFactory) {
         var beginningOfEra = DateFactory.createDate(0);
         var notEmpty = !Date.isSame(beginningOfEra);
 
-        if(Date.isValid() && notEmpty) return Date.format('DD/MM/YYYY');
+        if (!dateFormat) {
+            DateFormatFactory.getDateFormat().then(function (result) {
```

Comment:
How can I tell if the constant has already been initialised and has correct value, since there is no way of stoping the bootstrap of application until it's done?